src/views/project-board/ProjectBoard.vue

```diff
@@ -140,24 +147,26 @@ export default {
       changeView(viewKey);
     });
 
-    provide(
-      "shouldSubscribe",
-      computed(
-        () =>
-          currentSpace.value.isFree &&
-          currentSpace.value.isUserOrga &&
-          isFullTotal(spaceSubInfo.value)
-      )
+    useInterval(
+      async () => {
```

Comment:
useInterval only handles synchronous callbacks... async await not really useful here.